docs/bucket/notifications/README.md

```diff
@@ -424,8 +425,89 @@ key         |                      value
 (1 row)
 ```
 
+<a name="MySQL"></a>
+## Publish Minio events via MySQL
+
+Install MySQL from [here](https://dev.mysql.com/downloads/mysql/). For the purpose of illustrating sending events from Minio to MySQL, we set the root password as `password` and created a database called `miniodb` to store the events.
+
+### Step 1: Add MySQL server endpoint configuration to Minio
+
+The default location of Minio server configuration file is ``~/.minio/config.json``. Under the `notify` key, in the `mysql` sub-key, we may add as many MySQL server endpoint configurations as desired by providing an identifier for the server and an object of per-server configuration parameters.
+
+The per-server configuration parameters are shown in the following table.
+
+| Parameter | Value type | Description |
+|:---|:---|:---|
+| `enable` | Boolean | (Required) Is this server endpoint configuration active/enabled? |
+| `dsnString` | String | (Optional) [Data-Source-Name connection string](https://github.com/go-sql-driver/mysql#dsn-data-source-name) for the MySQL server. If not specified, the connection information specified by the `host`, `port`, `user`, `password` and `database` parameters are used. |
+| `table` | String | (Required) Table name in which events will be stored/updated. If the table does not exist, the Minio server creates it at start-up.|
+| `host` | String | Host name of the MySQL server (used only if `dsnString` is empty). |
+| `port` | String | Port on which to connect to the MySQL server (used only if `dsnString` is empty). |
+| `user` | String | Database user-name (used only if `dsnString` is empty). |
+| `password` | String | Database password (used only if `dsnString` is empty). |
+| `database` | String | Database name (used only if `dsnString` is empty). |
+
+An example of MySQL configuration is as follows:
+
+```
+"mysql": {
+        "1": {
+                "enable": true,
+                "dsnString": "",
+                "table": "minio_images",
+                "host": "172.17.0.1",
+                "port": "3306",
+                "user": "root",
+                "password": "password",
+                "database": "miniodb"
+        }
+}
+```
+
+Update the MySQL configuration block in your ``config.json`` similar to the above. Restart the Minio server to put the configuration changes into effect. The server will print a like like `SQS ARNs:  arn:minio:sqs:us-east-1:1:mysql` at start-up if the configuration validated successfully.
```

Comment:
Also, you can drop the `Update the MySQL configuration block in your ``config.json`` similar to the above` if you accept above suggestions